FordLabs/retroquest
# Replace svgs with font awesome

## What was done
Replaced the following SVG's with font-awesome icons
- The `Expired Password Link Page`, and the `Expired Board Owners Link Page` circle with exclamation point icon
- The `Password Input` eye icons

## Testing Instructions
1) Go to `http://localhost:3000/password/reset/expired-link` and ensure that the icon on the page is a font awesome icon and is the correct color in light and dark mode
2) Go to `http://localhost:3000/email/reset/expired-link` and ensure that the icon on the page is a font awesome icon and is the correct color in light and dark mode
3) `http://localhost:3000/login` and ensure the eye icon (both when the password is shown and hidden) is a font awesome icon and is the correct color when in light or dark mode.
